src/components/DynamicCard/DynamicCard.ts

```diff
@@ -118,6 +123,45 @@ async function getMarkup(element: DynamicCard) {
   return markup
 }
 
+/**
+ * Sets default values for DynamicCard attributes.
+ * These defaults will be applied to all DynamicCard instances created after this function is called.
+ *
+ * @param defaults - An object containing default values for DynamicCard attributes
+ *
+ * @example
+ * ```typescript
+ * import { setDynamicCardDefaults } from '@nosto/web-components'
+ *
+ * setDynamicCardDefaults({
+ *   placeholder: true,
+ *   lazy: true,
+ *   template: 'product-card'
+ * })
+ * ```
+ */
+export function setDynamicCardDefaults(
+  defaults: Partial<Pick<DynamicCard, "section" | "template" | "placeholder" | "lazy">>
+) {
+  dynamicCardDefaults = { ...defaults }
+}
+
+function applyDynamicCardDefaults(element: DynamicCard) {
+  // Only apply defaults if the corresponding attribute is not present in HTML
```

Comment:
generalize the applying of defaults into a Object.entries(...).forEach construct